File: app/src/main/java/org/neidhardt/dynamicsoundboard/soundmanagement/events/OnPlaylistChangedEventListener.java
package org.neidhardt.dynamicsoundboard.soundmanagement.events;

/**
 * File created by eric.neidhardt on 29.05.2015.
 */
public interface OnPlaylistChangedEventListener
{
	/**
	 * This is called by greenRobot EventBus in case the playlist has changed.*
	 * @param event delivered PlaylistChangedEvent
	 */
	void onEventMainThread(PlaylistChangedEvent event);
}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/soundmanagement/events/OnPlaylistChangedEventListener.kt
package org.neidhardt.dynamicsoundboard.soundmanagement.events

/**
 * File created by eric.neidhardt on 29.05.2015.
 */
public interface OnPlaylistChangedEventListener
{
	/**
	 * This is called by greenRobot EventBus in case the playlist has changed.*
	 * @param event delivered PlaylistChangedEvent
	 */
	@SuppressWarnings("unused")
	public fun onEventMainThread(event: PlaylistChangedEvent)
}
